pkg/healthcheck/healthcheck.go

```diff
@@ -2191,6 +2208,70 @@ func checkResources(resourceName string, objects []runtime.Object, expectedNames
 	return nil
 }
 
+func (hc *HealthChecker) checkMisconfiguredOpaquePortAnnotations(ctx context.Context, namespace string) error {
```

Comment:
wdyt about adding a small comment here to describe what misconfiguration means in this case?

Reply:
I've included a little comment; please, let me know if this makes things clearer?

Reply:
Yup! Comments look good 👌🏻 would you mind wrapping them at col 80 please? We try to keep most comments consistent across the codebase in terms of wrapping. Sorry, I should have mentioned it when I put the nitpicky feedback in.

Reply:
Sure thing!